offchain/dispatcher/src/machine/rollups_broker.rs

```diff
@@ -40,6 +39,13 @@ pub enum BrokerFacadeError {
 
     #[snafu(display("error consuming claim event"))]
     ConsumeClaimError { source: BrokerError },
+
+    #[snafu(whatever, display("{message}"))]
+    Whatever {
```

Comment:
Don't know if I missed it, but are we using this?

Reply:
If it is only being used for tests, I think we can put a `#[cfg(tests)]` on top of it so it won't compile for production.

Reply:
It's also being used inside the `dispatcher`, lines 109 and 120